src/dotnet/APIView/APIViewWeb/Client/webpack.config.js

```diff
@@ -47,7 +48,7 @@ module.exports = {
   },
   plugins: [
     new MiniCssExtractPlugin({
-      filename: 'site.css'
+      filename: '[name].css'
```

Comment:
This change will collide with [this change](https://github.com/Azure/azure-sdk-tools/pull/3978/files#diff-c2e1e48579187774acb6e65a024a9297216b3ffd1b97c2c390588a067cce1b3dR63). Once that PR is merged, you'll want to update this PR to make use of it.